getting-started/templates/systemlink-values.yaml

```diff
@@ -84,6 +84,32 @@ global:
     ## CDN endpoint for telemetry-driven content. NI does not recommend changing this value.
     ##
     cdnEndpoint: "https://web-sdk.ni.com/"
+  ##
+  ## MongoDB Database Configuration.
+  ## Systemlink Enterprise' various services utilize MongoDB databases for data storage. Customers can deploy their own MongoDB instance
+  ## or utilize a managed solution such as MongoDB Atlas. Once provisioned provide the connection details in the values below.
+  ##
+  mongodb:
+    ## Enable or disable installation of per-service MongoDB instances. 
+    #  <ATTENTION> - If you do *not* have your own MongoDB instance set this (`global.mongodb.install`) to `true`.
+    ##
+    install: false 
+    ## <ATTENTION> - If `global.mongodb.install` is false, configure the external MongoDB connection string to be used by services in this deployment.
+    #
+    # If you are setting credentials per service (e.g. assetservice.secrets.mongodb.servicePassword) leave the placeholders
```

Comment:
It seems like we should have a section in the ni.com documentation detailing all of these MongoDB choices. Is that already tracked work?

Reply:
I'm working on this [PR in Documentation Reviews](https://dev.azure.com/ni/Users/_git/Documentation%20Reviews/pullrequest/547599). Is there other documentation I should be tracking for changes?